app/src/main/AndroidManifest.xml

```diff
@@ -2,26 +2,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="openfoodfacts.github.scrachx.openfood"
-    android:installLocation="auto">
+    android:installLocation="auto"> <!-- Merging strategy for the imported manifests -->
+    <uses-sdk tools:overrideLibrary="io.sentry.android" />
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-
-    <!-- Merging strategy for the imported manifests -->
-    <uses-sdk tools:overrideLibrary="io.sentry.android" />
```

Comment:
Should sentry disappear ?

Reply:
@teolemon No, I formatted the code of **AndroidManifest**. So it's just some lines changed there position. If possible please check the line no. 6

Reply:
Please have a look at this screenshot.

![Untitled](https://user-images.githubusercontent.com/65870389/107643295-f6b94600-6c9b-11eb-8829-ea2144f341c4.png)